interface/src/app/gene/network/network.js

```diff
@@ -255,6 +244,22 @@ angular.module('adage.gene.network', [
         // any more.
       };
 
+      // Function that renders the sliders:
```

Comment:
This is a very minor comment, but I think this comment line is actually not necessary since the `renderSliders` function name is already descriptive enough! I think this comment just repeats that.

Reply:
Sure. I will do that.